src/device.js

```diff
@@ -77,7 +77,23 @@ var devicePixelRatio = window.originalDevicePixelRatio || window.devicePixelRati
 // @deprecated
 exports.devicePixelRatio = devicePixelRatio;
 
-exports.screen.devicePixelRatio = devicePixelRatio;
+var ctx = document.createElement('Canvas').getContext('2d');
```

Comment:
A couple minor changes here:
  * `exports.devicePixelRatio` should still be equal to `exports.screen.devicePixelRatio` despite the former being deprecated
  * use 2 spaces to indent and lines 86-91 are indented once too far

And just wondering, why use a getter here? Are some of the `window` properties used in the calculation asynchronously populated or changed dynamically?